docs/source/_static/js/loadShortbread.js

```diff
@@ -0,0 +1,27 @@
+// Load shortbread
+document.addEventListener('DOMContentLoaded', function() {
+
+	document.body.dataset.theme = localStorage.getItem("theme") || "auto";
+
+	if (typeof AWSCShortbread !== 'undefined') {
+		const shortbread = AWSCShortbread({
+        domain: ".amazonaws.com"
+        // domain: ".cloudfront.net"
```

Comment:
nit - remove this comment

Reply:
Add it as a comment with one sentence description instead of this.